TestIntegralConvergenceFV.py

```diff
@@ -111,7 +113,7 @@ def test2(lonRAD, latRAD):
        thisAvgs2 = np.zeros(numCells)
        thisAreas = np.zeros(numCells)
        for cc in range(numCells):
-              cdex = thisCell[cc,:] - 1
+              cdex = thisCell[cc,:] - 1 ######### NP.UNIQUE BUG #######
```

Comment:
Do you want this comment here?

Reply:
Yes, not relevant anymore. I'll clean it up.